src/null-safety/index.md

```diff
@@ -86,10 +76,58 @@ Dart null safety support is based on the following three core design principles:
   fewer bugs, but smaller binaries and faster execution.
 
 
-## Enabling/disabling null safety {#enable-null-safety}
+## Dart 3 and null safety
+
+Dart 3---planned for a mid-2023 release---requires sound null safety. It
+will prevent code from running without it.
```

Comment:
**Just a suggestion/my opinion:** Second sentence is confusing, two "It"s referring to different things.
```suggestion
Dart 3---planned for a mid-2023 release---requires sound null safety.
Dart 3 will prevent code from running without null safety.
```

Or if that's too repetitive:
```suggestion
Dart 3---planned for a mid-2023 release---requires sound null safety, and will prevent code from running without it.
```

@atsansone for a copy/style edit (idk how we feel about the em-dashed clause, maybe just parenthesis?)